encord/orm/dataset.py

```diff
@@ -499,6 +499,20 @@ def _update_current_class(self, new_class: DataRow) -> None:
         self.update(res_dict)
 
 
+@dataclasses.dataclass(frozen=True)
+class DataRows(dict, Formatter):
+    def __init__(self, data_rows: List[DataRow]):
+        super().__init__(
+            {
+                "data_rows": data_rows,
+            }
+        )
+
+    @classmethod
+    def from_dict(cls, json_dict: Dict) -> DataRow:
```

Comment:
interesting! I guess this is what is needed for `get_multiple`. I don't know a better way to solve this, we have similar hacks in other places. You can add a doc string to the class that it should not be used by users directly, but is more of an internal helper class.

Reply:
Yeah, that's the best I could do to call a separate service endpoint, while not introducing another public facing class and clutter the library API. Added the note in documentation